src/components/Column/Column.scss

```diff
@@ -229,6 +229,7 @@
   height: 24px;
   width: 24px;
   transition: all 0.08s ease-out;
+  transform: rotate(90deg);
```

Comment:
The icon is turned 90 degrees. This doesn't seem right 😄 
<img width="644" alt="Screenshot 2024-08-24 at 11 23 13" src="https://github.com/user-attachments/assets/06abb47e-818d-47a3-8c21-44e5d8ab9525">

Reply:
haha that was supposed to only affect the three dots icon:D just rotated the html element directly now